tests/test_userspace.py

```diff
@@ -163,3 +163,15 @@ def test_clocktree(clocktree, check, clock_name, rate, consumer):
     for c in consumer:
         with check:
             assert c in clk.consumer
+
+
+@pytest.mark.lg_feature("ptx-flavor")
+def test_ptx_ssh_keys(shell):
+    """
+    Check if there is at least one SSH key in the auto-generated authorized_keys.
+    This file is generated in our internal flavor of meta-lxatac and contains all relevant keys from our ansible.
+
+    The file starts with a comment, so we need to have at least two lines for the first key.
+    """
+    [lines] = shell.run_check("cat /etc/ssh/authorized_keys.root | wc -l")
+    assert int(lines) > 1
```

Comment:
I would suggest using `grep` to make sure there are actual keys instead of relying on the commented out header to have a specific length:

```suggestion
    shell.run_check('grep -q "^ssh-" /etc/ssh/authorized_keys.root')
```